Package.swift

```diff
@@ -3,7 +3,7 @@ import PackageDescription
 let package = Package(
     name: "VaporApp",
     dependencies: [
-        .Package(url: "https://github.com/vapor/vapor.git", majorVersion: 1, minor: 3)
+        .Package(url: "https://github.com/vapor/vapor.git", majorVersion: 1, minor: 4)
```

Comment:
That's a perfectly valid point. Let's see what a maintainer with write access thinks of it.